confidential-data-hub/storage/src/volume_type/mod.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2023 Intel
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#[cfg(feature = "aliyun")]
+pub mod alibaba_cloud_oss;
+
+#[cfg(feature = "aliyun")]
+use self::alibaba_cloud_oss::oss::Oss;
+use crate::{Error, Result};
+
+#[derive(PartialEq, Clone, Debug)]
+pub struct Storage {
+    pub driver: String,
+    pub driver_options: Vec<String>,
+    pub source: String,
+    pub fstype: String,
+    pub options: Vec<String>,
+    pub mount_point: String,
+}
+
+impl Storage {
+    pub async fn mount(&self) -> Result<String> {
+        for driver_option in &self.driver_options {
+            let (volumetype, metadata) =
+                driver_option
+                    .split_once('=')
+                    .ok_or(Error::SecureMountFailed(
+                        "split by \"=\" failed".to_string(),
+                    ))?;
+
+            match volumetype {
+                #[cfg(feature = "aliyun")]
+                "alibaba-cloud-oss" => {
+                    let oss: Oss = serde_json::from_str(metadata).map_err(|e| {
+                        Error::SecureMountFailed(format!(
+                            "illegal mount info format (json deseralization failed): {e}"
+                        ))
+                    })?;
+                    return oss
+                        .mount(self.source.clone(), self.mount_point.clone())
+                        .await;
+                }
+                other => {
+                    println!("skip mount info with unsupported volumetype: {other}");
```

Comment:
shall we use `log` here instead of `println!`? s.t. `log::info!` or `log::debug!`

Reply:
fixed, thanks